play/PlayerRules/Basic Rules.txt

```diff
@@ -407,192 +536,396 @@ Combat starts when either you or an enemy decides they would like to make an
 face-to-face unless you're at a bar.
-Generally speaking, once a character has initiated melee combat with another 
-character, they are both considered to be in "melee combat," and occupy 
-adjacent 1m spaces. At this point (unless one character has a CQB weapon) 
-both characters are assumed to be using their fists or other melee weapons. If
-a character wishes to leave melee combat, they incur a "penalty attack." 
-Characters also incur "penalty attacks" when moving through or next to an 
-enemy character''s 1m space, or within range of a melee weapon that the enemy 
-has equipped, and when performing a skill check when in melee combat.
-Melee combat is relatively simple. To hit an enemy player, add your dexterity 
-or strength modifier to a roll, plus any relevant skill. The result must be 
-greater than the opponent''s DEX * 5 + any skill in combat (Guard DC). Your 
-damage dealt is based on your DEX or STR modifier. If you are wielding a 
-weapon against an unarmed opponent, they may still incur damage depending on 
-their skill.
+	Generally speaking, once a character has initiated melee combat with another 
+character, they are both considered to be in "melee combat," and occupy adjacent 
+1m spaces. At this point (unless one character has a CQB weapon) both characters 
+are assumed to be using their fists or other melee weapons. If a character wishes 
+to leave melee combat, they incur a "penalty attack."  Characters also incur 
+"penalty attacks" when moving through or next to an enemy character's 1m space,
+or within range of a melee weapon that the enemy has equipped and when performing 
+a skill check when in melee combat.
+	Melee combat is relatively simple. To hit an enemy player: add your dexterity 
+combat modifier to a d10 roll plus your combat modifier from your weapon/attack*,
+and add any relevant skill(s)**. The result must be greater than the opponent's
+Guard DC: (DEX * 4)/10 + any skill** in combat. Drop any decimal remainder. If you
+successfully hit your opponent, you roll armor as normal (for more details, see below sections 
+"Armor and Getting Hit" and "Damage Types and Armor Types"). Your damage dealt is often
+based on a combination of your DEX and STR. If you are wielding a weapon against
+an unarmed opponent, they may still incur damage depending on their skill.
+	NOTE: if you are fighting unarmed, you can (normally) deal full non-lethal damage or 
+half lethal damage (your choice). Non-lethal damage should be recorded as a separate (duplicate) 
+health bar. When their non-lethal health bar reaches 0, the target becomes unconscious/incapacitated.
+	*Modifiers from specific weapons or unarmed[x] attacks are listed in the weapons document.
+	**To determine bonus modifiers from skill, for every 5 skill points in a relevant 
+	skill (such as "melee combat skill") take a bonus +0.5 when rolling against guard DC.
+	Drop any decimal remainder after adding all the modifiers.

```

Comment:
It's so nice to have melee rules I feel like I can actually understand enough to run. And they're all even written down!

Reply:
:D